pkg/apis/componentconfig/edgecore/v1alpha1/default.go

```diff
@@ -283,3 +286,31 @@ func NewEdgeClusterEdgeCoreConfig() *EdgeCoreConfig {
 		},
 	}
 }
+
+//SetKubeconfigArray returns an array of kubeconfig files
+func SetKubeconfigArray() []string {
+	dir, err := ioutil.ReadDir("/etc/fornax/configs/")
+	if err != nil {
+		log.Fatal(err)
+	}
+	var configFiles []string
+	for _, file := range dir {
+		if strings.ContainsAny(file.Name(), ".") {
+			if strings.Split(file.Name(), ".")[1] == "kubeconfig" {
+				configFiles = append(configFiles, file.Name())
+			}
+		}
+	}
+	return configFiles
+}
+
+//ToGivenCluster returns true if kubeconfig file exists in the current edgecluster.
+func ToGivenCluster(input string) bool {
```

Comment:
this name is a bit confusing. it could be renamed to "ConfigFileExist" or something like that

Reply:
Updated name of the function, thanks